src/coreclr/interpreter/interpretershared.h

```diff
@@ -18,12 +18,14 @@ struct InterpMethod
     CORINFO_METHOD_HANDLE methodHnd;
     int32_t allocaSize;
     void** pDataItems;
+    uint32_t initLocals : 1;
```

Comment:
nit: Feels somewhat weird to use the specific `uint32_t` for a bitfield value. Seems cleaner to use a `bool` instead.